source/libs/api.ts

```diff
@@ -67,15 +74,16 @@ function fetch4(query, personalToken) {
 }
 
 // Main function: handles cache, options, errors
-async function call(fetch, query, options = {accept404: false}) {
+async function call<T extends SoManyFetch>(fetch: T, query: string, options = {accept404: false}) {
 	if (cache.has(query)) {
 		return cache.get(query);
 	}
 
-	const {personalToken} = await new OptionsSync().getAll();
-	const response = await fetch(query, personalToken);
+	const {personalToken} = await new OptionsSync().getAll<{personalToken: string}>();
+	const response = await fetch4(query, personalToken);
 	const content = await response.text();
-	const result = content.length > 0 ? JSON.parse(content) : {status: response.status};
+
```

Comment:
I made these optional because, they won't exist if `content.length` is zero. Also, when it is zero, another object was created, that wasn't used,` { status: response.status }`, so I made it an empty object instead.

Reply:
It is used here: https://github.com/sindresorhus/refined-github/blob/d01f4d70daddbd03fa94e3d1b2009bf84ef6eee9/source/features/user-profile-follower-badge.tsx#L8

Reply:
Put this back so it works still for `const {status} = await api.v3(...`